### ISSUE TITLE
hipchat notification module doesn't properly type check room ID

### ISSUE THREAD

**Issue by zacharya** [ansible/ansible-modules-extras#1381]
I was running into an issue when trying to use a room ID (int) versus a room name (string):

``` YAML

---
  - hipchat:
    api: "https://our.hipchat.server/v2"
    validate_certs: yes
    token: "token"
    room: 99
    msg: "message to lobby"
```

``` python
ASK: [notify hipchat] ********************************************************
failed: [host1] => {"failed": true}
msg: unable to send msg: expected a character buffer object
```

If I didn't quote the room ID (which one would expect to be an int), I found that when main loads the parameters via the AnsibleModule, it doesn't do any type checking.  So if a user passes an unquoted room ID:

``` python
room = module.params["room"]
```

 an int can make it through to the send_msg methods (in my case send_msg_v2):

``` python
try:
    if api.find('/v2') != -1:
        send_msg_v2(module, token, room, msg_from, msg, msg_format, color, notify, api)
    else:
        send_msg_v1(module, token, room, msg_from, msg, msg_format, color, notify, api)
except Exception, e:
    module.fail_json(msg="unable to send msg: %s" % e)
```

**Comment by zacharya** [comment]
Submitted #1380 to fix.

**Comment by zacharya** [comment]
Fixed with #1380
